test/functional/testlib.bash

```diff
@@ -2395,10 +2399,12 @@ create_bundle() { # swupd_function
 		return
```

Comment:
you need to change this call to the help menu here (line 2398), at the moment is using `create_bundle -h` to display the help menu, since there was no `h` parameter it was just defaulting to showing the help menu, but since you are defining that parameter then we need to call the help menu by just calling `cb_usage`